RangeHTTPServer.py

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/python
 '''
-Use this in the same way as Python's SimpleHTTPServer:
+Use this in the same way as Python's http.server:
```

Comment:
How about `http.server / SimpleHTTPServer`? It works with either Python 2 or 3, after all.